include/mbedtls/config_adjust_psa_superset_legacy.h

```diff
@@ -70,4 +70,98 @@
 #define PSA_WANT_ALG_SHA3_512 1
 #endif
 
+/* Ensure that the PSA's supported curves (PSA_WANT_ECC_xxx) are always a
+ * superset of the builtin ones (MBEDTLS_ECP_DP_xxx). */
+#if defined(MBEDTLS_ECP_DP_BP256R1_ENABLED)
+#define MBEDTLS_SOME_BUILTIN_EC
```

Comment:
Looking again, this file should not define `MBEDTLS_SOME_BUILTIN_EC`. The purpose of this file is to enable certain PSA mechanisms if the corresponding legacy mechanism is enabled. So it should only have conditionals based on public `MBEDTLS_xxx` symbols (or in a stretch, derived internal `MBEDTLS_xxx` symbols), and it should only define `PSA_WANT_xxx` symbols.

Whether to define `MBEDTLS_SOME_BUILTIN_EC` should only depend on `PSA_WANT_xxx` and `MBEDTLS_PSA_ACCEL_xxx`, regardless of how those were set.